src/analysis/special_functions/compare_exp.lean

```diff
@@ -0,0 +1,182 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import analysis.special_functions.pow
+import analysis.asymptotics.specific_asymptotics
+
+/-!
+# Growth estimates on `x ^ y` for complex `x`, `y`
+
+Let `l` be a filter on `ℂ` such that `complex.re` tends to infinity along `l` and `complex.im z`
+grows at a subexponential rate compared to `complex.re z`. Then
+
+- `complex.is_o_log_abs_re_of_subexponential_im_re`: `real.log ∘ complex.abs` is `o`-small of
+  `complex.re` along `l`;
+
+- `complex.is_o_cpow_mul_exp`: $z^{a_1}e^{b_1 * z} = o\left(z^{a_1}e^{b_1 * z}\right)$ along `l`
+  for any complex `a₁`, `a₂` and real `b₁ < b₂`.
+
+We use these assumptions on `l` for two reasons. First, these are the assumptions that naturally
+appear in the proof. Second, in some applications (e.g., in Ilyashenko's proof of the individual
+finiteness theorem for limit cycles of polynomial ODEs with hyperbolic singularities only) natural
+stronger assumptions (e.g., `im z` is bounded from below and from above) are not available.
+
+-/
+
+open asymptotics filter function
+open_locale topological_space
+
+namespace complex
+
+/-- We say that `l : filter ℂ` is an *exponent comparison filter* if the real part tends to infinity
```

Comment:
`exponential comparison filter`, maybe? (As the goal is to say something about the exponential function).